praw/models/reddit/subreddit.py

```diff
@@ -487,6 +487,14 @@ def submit(self, title, selftext=None, url=None, flair_id=None,
             data.update(kind='link', url=url)
         return self._reddit.post(API_PATH['submit'], data=data)
 
+    def _crosspost(self, fullname, title=None, send_replies=True):
```

Comment:
I think this method should live in the submission rather than on the subreddit.

Reply:
Hmm, I was thinking about using this method to promote DRYness with the `data` dictionary since comments will also be crosspostable in the future. If we moved it to the Submission class, the Comment class wouldn't be able to access it unless if it's a static method right?

Reply:
I like your forward thinking. If this method can be used on both comments and submissions, then I would actually recommend putting it in to a Mixin type class like `MessagableMixin` is (inclusion: https://github.com/praw-dev/praw/blob/092f24467228588672ee4dd37c1051ccd8f8732e/praw/models/reddit/subreddit.py#L19, source: https://github.com/praw-dev/praw/blob/092f24467228588672ee4dd37c1051ccd8f8732e/praw/models/reddit/mixins/messageable.py).